storage/myisam/mi_check.c

```diff
@@ -1225,9 +1205,9 @@ int chk_data_link(HA_CHECK *param, MI_INFO *info, my_bool extend)
                            SEARCH_SAME, info->s->state.key_root[key]);
               if (search_result)
               {
-                mi_check_print_error(param,"Record at: %10s  "
+                mi_check_print_error(param, "Record at: %10lld  "
                                      "Can't find key for index: %2d",
-                                     llstr(start_recpos,llbuff),key+1);
+                                     start_recpos, key+1);
```

Comment:
this is one of the errors, I suppose, that can be reported many times for different records. So it kind of makes sense to align the offset here